third_party/material/initialize.sh

```diff
@@ -14,6 +14,7 @@ EOF
 
 npm install "@material/auto-init@${VERSION}"
 npm install "@material/button@${VERSION}"
+npm install "@material/fab@${VERSION}"
```

Comment:
Don't we also need a JS initialization for this? Maybe not, as I couldn't find any traces here:
https://github.com/material-components/material-components-web/blob/master/packages/material-components-web/index.ts

Reply:
I think not. In my example it seems to work without initialization.